include/deal.II/dofs/dof_handler.h

```diff
@@ -1098,7 +1094,8 @@ class DoFHandler  :  public Subscriptor
 #ifndef DOXYGEN
 
 template <> types::global_dof_index DoFHandler<1>::n_boundary_dofs () const;
-template <> types::global_dof_index DoFHandler<1>::n_boundary_dofs (const FunctionMap &) const;
+//template <> template <> types::global_dof_index DoFHandler<1>::n_boundary_dofs<double> (const FunctionMap<double> &) const;
+//template <> template <> types::global_dof_index DoFHandler<1>::n_boundary_dofs<float> (const FunctionMap<float> &) const;
```

Comment:
i am not sure if these are needed or not. If i understand it correctly, it's a promise that there will be an explicit specializations for those cases. But if I commeit it out while keeping the specialized versions in .cc file it must be all good, right?